pingdynasty/OwlProgram
# adjustments for new hvcc

Seems to work for me with these changes (and you can now drop python2.7 requirements).
I am probably missing some things since this entire process seems quite involved.
(my editor also removed trailing white-space in the README btw)

What I did to test this:
* install emsdk and source emsdk_env.sh in the current shell (for emcc requirement)
* `make libs`
* build FirmwareSender and put it in Tools/
* `make HEAVY=Lich_PD_template load`

You are probably not ready to switch to this, but at least should give some hints on what changes are needed to transition.
